client/components/BarRating.jsx

```diff
@@ -47,8 +47,14 @@ const Rating = styled.span`
   width: ${(props) => ((props.rating / 5) * 100)}%;
 `;
 
-const BarRating = ({ className, rating, clean }) => (
-  <div className={className}>
+const BarRating = ({
+  className,
+  rating,
+  clean,
+  clickable,
+  ...rest
+}) => (
+  <div className={className} clickable={clickable ? 1 : 0} {...rest}>
     <List>
       {clean ? <></> : (
```

Comment:
I've often preferred `{ clean && ... }` instead of ternaries here, since the `...` will only render if `clean` is true. (This way, I don't have to add more code to say "don't render anything if `clean` is false.")